easybuild/easyconfigs/m/magma/magma-2.5.4-fosscuda-2019b.eb

```diff
@@ -15,15 +15,15 @@ sources = [SOURCE_TAR_GZ]
 patches = ['magma-2.5.4_allow-all-sms.patch']
 checksums = [
     '7734fb417ae0c367b418dea15096aef2e278a423e527c615aab47f0683683b67',  # magma-2.5.4.tar.gz
-    '0de5d7b3de7f53e54aeb309193cce04a6a5b265a494dc5416e18fd7d6734386e',  # magma-2.5.4_allow-all-sms.patch
+    '4e47ecc080ff761866f179c9b383b8930e4daa2fcee1c2a48db585fc50531561',  # magma-2.5.4_allow-all-sms.patch
 ]
 
 builddependencies = [
     ('CMake', '3.15.3'),
 ]
 
 # default CUDA compute capabilities to use (override via --cuda-compute-capabilities)
-cuda_compute_capabilities = ['3.0', '3.5', '5.0', '6.0', '7.0', '7.5']  # or 'Kepler Maxwell Pascal Volta Turing'
+cuda_compute_capabilities = ['3.0', '3.5', '5.0', '6.0']  # or 'Kepler Maxwell Pascal Volta'
```

Comment:
I don't you intended to remove 7.0 and 7.5 again?

Reply:
Woops. update-pr is tricky, thanks!